jazz-build-module/service-metadata-loader.groovy

```diff
@@ -66,6 +66,27 @@ def loadServiceMetaData() {
 	}
 }
 
+/**
+ * Load the service metadata from Catalog
+ *
+ */
+ 
+def loadServiceMetaDataByServiceId() {
```

Comment:
The signature for this method seems wrong. We should pass serviceId as a param. Once you add that as param, update the method to say loadServiceMetaData

Reply:
Not clear

Reply:
The method should me **loadServiceMetaData**(_serviceId_)

Reply:
The loadServiceMetaData() method is calling /platform/services with Query Params. The loadServiceMetaDataByServiceId() method is calling /platform/services/{service_id}. The response also different. Thats why kept two separate methods.

Reply:
In cooperated review comments